crates/ruff_python_formatter/src/statement/stmt_global.rs

```diff
@@ -1,12 +1,18 @@
-use crate::{not_yet_implemented, FormatNodeRule, PyFormatter};
-use ruff_formatter::{write, Buffer, FormatResult};
+use ruff_formatter::{format_args, write, Buffer, FormatResult};
```

Comment:
```suggestion
use ruff_formatter::{format_args, write};
```

You should get these for free when importing the prelude.